### DISCUSSION TITLE
⚡️Request for Woo Discount Rules plugin⚡️

### DISCUSSION THREAD

**Discussion by Uhtredtb** [inspireui/support#5405] [New Feature/Ideas Requests]
It would be awesome for the app to support this Woo Discount Rules plugin so that all the rules can be applied to the app and the web similarly.

Plugin URL: https://wordpress.org/plugins/woo-discount-rules/
Pro Version: https://www.flycart.org/products/wordpress/woocommerce-discount-rules

Please consider thanks.
Anyone who agrees, please join my comment :)

**Comment by ShubhuSR** [comment]
Yes, this feature is must needed

**Comment by QuickBasket** [comment]
Yes discounts plugins displaying same Information in web and App are must.
another good option is: 

https://algolplus.com/plugins/downloads/advanced-dynamic-pricing-woocommerce-pro/

I particularly like it because it also offers my inventory based discounts as well.

**Comment by QuickBasket** [comment]
A woocomerce discount plugin integration would help a lot. hope this has been looked into, Would request others to support a discount plugin integration suggestion.

**Comment by tjain90** [comment]
Completely agree. This is must needed feature.

**Comment by QuickBasket** [comment]
hi any news on this?

**Comment by JayDeeZzz** [comment]
Hi guys, this is a much needed feature. Because adding the sale price on more than 100 products are really a pain in the A**. So hope you guys fix this issue asap 😊 Cheers.....

**Comment by medtr-git** [comment]
Hi,
I think it's the most important feature right now.
To developer, please consider taking a look at this one
